nzedb/Binaries.php

```diff
@@ -724,27 +748,33 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 					// Get the current unixtime from PHP.
 					$now = time();
 
+					$xref = ($multiGroup === true ? sprintf('xref = CONCAT(xref, "\\n"%s ),', $this->_pdo->escapeString(substr($header['Xref'], 2, 255))) : '');
```

Comment:
This still feels kinda dirty to me. XREF is data that comes from the usenet server, we shouldn't change it, any changes needed should go somewhere else so that the original value is always correct.